README.md

```diff
@@ -6,6 +6,62 @@ The current set of extensions can be read on [GitHub Pages](https://ocfl.github.
 
 See also [pending pull requests](https://github.com/OCFL/extensions/pulls) for extensions under discussion.
 
+## Implementing Community Extensions
+
+Reference the spec's description of [object extensions](https://ocfl.io/1.0/spec/#object-extensions) and [storage root extensions](https://ocfl.io/1.0/spec/#storage-root-extensions).
+
+A *root extension directory* refers to the directory named `extensions` that is located in either the storage root or an object root. An *extension directory* is an extension specific directory that is the child of a root extension directory and named using the extension's *Registered Name*. For example, `extensions/0000-example-extension` is the extension directory for the extension [0000-example-extension](docs/0000-example-extension.md).
+
+Each extension specification details how it should be implemented, but there are a few rules that apply to every extension.
+
+The OCFL storage root MAY contain a copy of an extension's specification.
+
+### Manifest
+
+Every extension that is intended to be used MUST be declared in an extension manifest file. The extension manifest file is a JSON file named `extensions.json` that is a child of a root extension directory. This file MUST contain a JSON object with a single key, `extensions`, thats value is an array of registered extension names.
+
+For example, using the extension [0000-example-extension](docs/0000-example-extension.md) in an object, necessitates creating a manifest file at `OBJECT_ROOT/extensions/extensions.json` that contains the following:
+
+```json
+{
+  "extensions": [
+    "0000-example-extension"
+  ]
+}
+```
+
+Storage layout extensions MUST NOT be declared in the manifest. They are instead declared in `ocfl_layout.json`, as described in the [OCFL spec](https://ocfl.io/1.0/spec/#root-structure).
+
+It is permissible for a root extension directory to contain extension directories for extensions that are not declared in the extension manifest. Extensions that are not declared MUST NOT be used.
```

Comment:
"It is permissible for a root extension directory to contain" -> "A root extension directory MAY contain.."